### PR TITLE
feat: Determine whether to log out according to the 401 and error messages on the member cluster page

### PR DESCRIPTION
**Pull request by harrisonliu5**
Signed-off-by: harrisonliu5 <harrisonliu_5@163.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

 /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #[#kubesphere/kubesphere/3935](https://github.com/kubesphere/kubesphere/issues/3935)

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```
/cc @wansir

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -517,6 +517,22 @@ export const withDryRun = async requests => {
 export const isAppsPage = (path = location.pathname) =>
   path === '/apps' || path.startsWith('/apps/app-')
 
+export const isMemberClusterPage = (path = location.pathname, message) => {
+  const member = path.indexOf('/clusters') > -1 ? path.split('/')[2] : 'host'
```
Comment (wansir) [inline src/utils/index.js:521]: Precise matching is needed, similar problem: https://github.com/kubesphere/kubesphere/issues/3975

Comment (harrisonliu5) [reply src/utils/index.js]: updated

**Pull request merged**